src/rtcModule/webrtc.cpp

```diff
@@ -133,13 +133,20 @@ void RtcModule::initInputDevices()
     }
 }
 
-void RtcModule::removeCallRetry(karere::Id chatid)
+void RtcModule::removeCallRetry(karere::Id chatid, bool failed)
```

Comment:
Perhaps it's easier to understand if we refactorize this variable into `retry` (being the opposite value than now).

Reply:
Fixed it